lib/Runtime/Library/DateImplementation.cpp

```diff
@@ -1266,8 +1266,17 @@ namespace Js {
                 goto LError;
             }
 
-            for (lwT = ch - '0'; !FBig(*pch) && isdigit(*pch); pch++)
+            for (lwT = ch - '0'; ; pch++)
             {
+                // for time zone offset HH:mm, we already got HH so skip ':' and grab mm
```

Comment:
Ok, it took a look at the math below to understand how this works because this just accumulates the HH:mm values so `-07:30` just becomes `730`. Below we convert this into minutes, correct? Would you mind adding a small comment somewhere close to here explaining that?

Reply:
@boingoing yes, it is correct. Will do.

Reply:
Great, thanks